Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN yum install -y python-requests && \
     yum update -y && \
     yum install -y openstack-ironic-api openstack-ironic-conductor crudini \
         iproute iptables dnsmasq httpd qemu-img-ev iscsi-initiator-utils parted gdisk ipxe-bootimgs psmisc sysvinit-tools \
-        mariadb-server python-PyMySQL python2-chardet && \
+        mariadb-server python2-chardet && \
```

Comment:
I wonder if we should also remove python2-chardet, as I assume this is also wrong?

https://github.com/openshift/ironic-image/blob/master/Dockerfile.ocp#L6

Reply:
python2-chardet is correct